src/index.html

```diff
@@ -1,13 +1,391 @@
+    >
+    <title>Miami landing page</title>
+    <link
+      rel="icon"
+      type="image/png"
+      href="./images/favicon.png"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin=""
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="header page__section">
+      <div class="header__container">
+        <nav class="top-actions">
+          <a class="icon icon--menu" href="#menu"></a>
+          <a class="logo" href="#">
+            <img
+              class="logo__img"
+              src="./images/logo.png"
+              alt="Miami Condo Kings"
+            >
+          </a>
+          <a href="tel:+1 234 555-55-55" class="icon icon--tel"></a>
+        </nav>
+        <h1 class="header__title">
+          Miami
+          <br>
+          Condo Kings
+        </h1>
+      </div>
+    </header>
+    <nav class="page__menu menu page__body--with-menu" id="menu">
+      <div class="menu__container">
+        <nav class="top-actions">
+          <a class="icon icon--cross" href="#"></a>
+          <a class="logo" href="#">
+            <img
+              class="logo__img"
+              src="./images/logo.png"
+              alt="Miami Condo Kings"
+            >
+          </a>
+        </nav>
+        <ul class="menu__list">
+          <li class="menu__item">
+            <a href="#" class="menu__link">Home</a>
+          </li>
+          <li class="menu__item">
+            <a href="#about-us" class="menu__link">About us</a>
+          </li>
+          <li class="menu__item">
+            <a href="#gallery" class="menu__link">Gallery</a>
+          </li>
+          <li class="menu__item">
+            <a href="#testimonials" class="menu__link">Testimonials</a>
+          </li>
+          <li class="menu__item">
+            <a href="#contacts" class="menu__link">Contacts</a>
+          </li>
+        </ul>
+        <a
+          href="tel:+1 234 555-55-55"
+          class="menu__tel menu__tel--tel"
+        >
+        +1 234 555-55-55
+        </a>
+        <a
+          href="tel:+1 234 555-55-55"
+          class="menu__tel menu__tel--action"
+        >
+        call to order
+        </a>
+      </div>
+    </nav>
+    <main class="page__main">
+      <section class="page__section about" id="about-us">
+        <div class="container">
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--1-4">
+              <h2 class="about__title">
+                Miami best
+                <br>
+                Real estate
+                <a href="#contacts" class="about__title-link link">here</a>
```

Comment:
Here should be link FROM contact-us not TO contact-us block, that I mean https://goo.gl/maps/wjgcBHEhtZ7M8wJA7